.circleci/config.yml

```diff
@@ -107,7 +107,9 @@ commands:
           command: |
             if [ ${CIRCLE_BRANCH} != "" ] &&
               [ ${CIRCLE_BRANCH} != "master" ] &&
-              ! git diff --name-only $(git merge-base origin/master ${CIRCLE_BRANCH}) | grep -qvE '(.md$)|(.pdf$)'; then
+              ! git diff --name-only $(git merge-base origin/master ${CIRCLE_BRANCH}) \
+                | grep -qvE '(.md)|(.pdf)|(.png)|(.svg)|(Test.java)|(Test.kt)|(.mp4)|(.log)|(.txt)'; then
```

Comment:
puppeteer job calls this. It doesn't make sense to me to run puppeteer tests if commits include only tests. Since there is no product code change, I think running puppeteer tests would be waste of time and resources.